### DISCUSSION TITLE
Google's "I'm Feeling Lucky" feature

### DISCUSSION THREAD

**Discussion by iitjee** [infokiller/web-search-navigator#395] [General]
There is a feature on Google called "I'm feeling lucky" which direclty takes you to the first result, skipping the results page. Would love to see such a feature in this extension as well. Shouldn't be very difficult to implement I guess?

PS: Hugely appreciate your work. Easily one of my favorite extensions. Kudos!

**Comment by infokiller** [comment]
Shouldn't be difficult to implement, with the caveat that the extension will need to navigate to the search results page and only then navigate to the first result. I guess there is some complexity to understand whether the first result also applies to cards, inline answers and stuff like that.

I welcome PRs that implement this, but I personally don't have much interest in this feature, because navigating to the first result is only a space/enter away, and I rarely want to blindly select it.
